lib/docs/src/tag.stories.tsx

```diff
@@ -1,24 +1,42 @@
 import { Meta } from "@storybook/react";
 import { Fragment } from "react";
-import { DivPx, Tag } from "../../core/src";
+import { Button, DivPx, Tag } from "../../core/src";
+import { Utils } from "./utils";
+
+const colors: string[] = [
+	"red",
+	"yellow",
+	"green",
+	"blue",
+	"indigo",
+	"purple",
+	"pink",
+	"gray",
+];
 
 export default {
-	title: "Draft/Tag",
+	title: "Components/Tag",
 	component: Tag,
+	argTypes: {
+		children: Utils.arg("text"),
+		color: Utils.arg(colors),
+	},
 } as Meta;
 
-export const Primary = (): JSX.Element => {
-	const colors: string[] = [
-		"red",
-		"yellow",
-		"green",
-		"blue",
-		"indigo",
-		"purple",
-		"pink",
-		"gray",
-	];
+interface Props {
+	children?: string;
+	color?: string;
+}
+
+export const Primary = (props: Props): JSX.Element => {
+	return (
+		<Tag color={(Tag.colors as any)[props.color ? props.color : "gray"]}>
+			{props.children ? props.children : "Default"}
```

Comment:
A shorter way is:
`{props?.children || "Default"}`

Reply:
@lqt93 and "props" is always defined, so I think we don't need that "?"

and also I think "??" is better than "||" in this case

Reply:
My fault. So, it is good to use
`props.children ?? "Default`